packages/block-editor/src/components/inspector-controls/README.md

```diff
@@ -374,4 +179,44 @@ registerBlockType( 'my-plugin/inspector-controls-example', {
 } );
 ```
 
-{% end %}
+## InspectorAdvancedControls
```

Comment:
Let's keep this section in the documentation until we mark the group as a stable API.